app/__init__.py

```diff
@@ -37,29 +36,44 @@
     }
 )
 
-# Route for creating workspaces
+
 @app.route('/workspace/create', methods=['POST'])
-def linchpin_init():
+def linchpin_init() -> Response:
```

Comment:
@mansikulkarni96 is this correct syntax ??

Reply:
As of python 3.5 it is the syntax to mention method return type:
https://stackoverflow.com/questions/5336320/how-to-know-function-return-type-and-argument-types/31152839

Reply:
Surprising I have not used this sytax yet . 
I better note it down :)